components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -1977,10 +1981,14 @@ def getter(attr):
                 accessor = 'get_' + self.descriptor.internalNameFor(attr.identifier.name)
                 jitinfo = "0 as *const JSJitInfo"
             else:
+                if attr.type.isPromise():
+                    exceptionToRejection = "true"
+                else:
+                    exceptionToRejection = "false"
```

Comment:
use str here? `str(attr.type.isPromise()`

Reply:
Python has uppercase `True`/`False`, we could use `str(True).lower()`, but I think that would be to obfuscated. Also ideally we would use enum here, but cons generics does not support it yet.